### PR TITLE
Revert "No more wearable collars (#20440) and renames them to just "Collar" "

### PR DESCRIPTION
**Pull request by Hislyna**
This reverts commit 54428f133b52d34fac8551f206b1ca9929f6b78c.

# Document the changes in your pull request

Makes it so you can wear collars in the game again.

# Why is this good for the game?
Stop removing things cause a few people say its sexual. You can literally run around naked with your tits out in this game, this is nothing. Also removed the pet so all fetish talk on that bit will stop.

# Testing
Yea it works

# Changelog

:cl:  
rscadd: Changes pet collars back to the way it was before and calls it just "Collar"
/:cl:

### CODE REVIEW

**Comment by azzzertyy** [comment]
ok, your why doesnt list any reason that this SHOULD be in the game.
how does this benefit the game?

**Comment by Hislyna** [comment]
> ok, your why doesnt list any reason that this SHOULD be in the game. how does this benefit the game?

This is a revert, not a new feature.

**Comment by azzzertyy** [comment]
> > ok, your why doesnt list any reason that this SHOULD be in the game. how does this benefit the game?
> 
> This is a revert, not a new feature.

and? it was removed for a reason, you havent given any justification why it shouldnt be removed other than "bad change!"

**Comment by Hislyna** [comment]
> > > ok, your why doesnt list any reason that this SHOULD be in the game. how does this benefit the game?
> > 
> > 
> > This is a revert, not a new feature.
> 
> and? it was reverted for a reason, you havent given any justification why it shouldnt be reverted other than "bad change!"

The justification is that the justification for it being removed is poor. There is nothing wrong with having collars as a clothing item. Chokers are fine.

**Comment by Moltijoe** [comment]
> The justification is that the justification for it being removed is poor. There is nothing wrong with having collars as a clothing item. Chokers are fine.

Cokers are not the same as collars, certainly not pet collars, feel free to add a choker if so you desire

**Comment by Hislyna** [comment]
> > The justification is that the justification for it being removed is poor. There is nothing wrong with having collars as a clothing item. Chokers are fine.
> 
> Cokers are not the same as collars, certainly not pet collars, feel free to add a choker if so you desire

Theres nothing sexual about the collar, I see noone using it as a fetish item, and it is just a few people crying about it. If someone did use it sexually, they would have been banned under 0.2

**Comment by Moltijoe** [comment]
Eh fine, whatever, but rename it from pet collar to collar

**Comment by Hislyna** [comment]
> Eh fine, whatever, but rename it from pet collar to collar

Edited! (Bet I broke it lol)

**Comment by wonderinghost** [comment]
(ya didn't)

**Comment by JamieD1** [comment]
Its not sexual its cringe

**Comment by Hislyna** [comment]
> Its not sexual its cringe

That ill accept completely as facts.

**Comment by MajManatee** [comment]
while it's not INHERENTLY fetish content, the people who wear them intend for them to be used as such. if you want a dedicated "choker" item to satisfy the white girl Wednesday vibe inside then please feel free to add one, hell ill code it for you. Changing the name does nothing besides trying to sidestep the fetish rule. 
If we can't have a pie stain in a maint room, then why would this be allowed?

**Comment by SomeguyManperson** [comment]
Yogmins when the ability to wear a collar labeling you "x's pet" is removed

**Comment by N3D6** [comment]
Rather die than have this back in the game honestly

the use of this was inherently horny and will be the exact same should it be re-added
Tldr touch grass

**Comment by Hislyna** [comment]
> while it's not INHERENTLY fetish content, the people who wear them intend for them to be used as such. if you want a dedicated "choker" item to satisfy the white girl Wednesday vibe inside then please feel free to add one, hell ill code it for you. Changing the name does nothing besides trying to sidestep the fetish rule. If we can't have a pie stain in a maint room, then why would this be allowed?

Personally I think it was childish that the pie stain was removed again and should be readded, but at the same time, it is completely different to this.



> Rather die than have this back in the game honestly
> 
> the use of this was inherently horny and will be the exact same should it be re-added Tldr touch grass

The inherent use is not horny. Report those who do use it for that as a 0.2 violation. Its just a collar. The fact yall are getting so upset about this is a joke. Stop being children about an item that doesn't effect you.

**Comment by N3D6** [comment]
> > while it's not INHERENTLY fetish content, the people who wear them intend for them to be used as such. if you want a dedicated "choker" item to satisfy the white girl Wednesday vibe inside then please feel free to add one, hell ill code it for you. Changing the name does nothing besides trying to sidestep the fetish rule. If we can't have a pie stain in a maint room, then why would this be allowed?
> 
> Personally I think it was childish that the pie stain was removed again and should be readded, but at the same time, it is completely different to this.
> 
> > Rather die than have this back in the game honestly
> > the use of this was inherently horny and will be the exact same should it be re-added Tldr touch grass
> 
> The inherent use is not horny. Report those who do use it for that as a 0.2 violation. Its just a collar. The fact yall are getting so upset about this is a joke. Stop being children about an item that doesn't effect you.

what part of “this is horny and stupid don’t put it in the game” makes me childish lol this is horny and stupid don’t put it in the game

**Comment by Hislyna** [comment]
> > > while it's not INHERENTLY fetish content, the people who wear them intend for them to be used as such. if you want a dedicated "choker" item to satisfy the white girl Wednesday vibe inside then please feel free to add one, hell ill code it for you. Changing the name does nothing besides trying to sidestep the fetish rule. If we can't have a pie stain in a maint room, then why would this be allowed?
> > 
> > 
> > Personally I think it was childish that the pie stain was removed again and should be readded, but at the same time, it is completely different to this.
> > > Rather die than have this back in the game honestly
> > > the use of this was inherently horny and will be the exact same should it be re-added Tldr touch grass
> > 
> > 
> > The inherent use is not horny. Report those who do use it for that as a 0.2 violation. Its just a collar. The fact yall are getting so upset about this is a joke. Stop being children about an item that doesn't effect you.
> 
> what part of “this is horny and stupid don’t put it in the game” makes me childish lol this is horny and stupid don’t put it in the game

The fact that your only arguement is "Oh its horny and sexual!!!" when its not proves my point.

**Comment by azzzertyy** [comment]
> The fact that your only arguement is "Oh its horny and sexual!!!" when its not proves my point.

i may have missed something but what exactly is your argument his? the only one ive seen is "shouldnt have been removed" which isnt really an argument for it

**Comment by Necrosium** [comment]
Eh whatever as long as the fetish stuff is off the table...

**Comment by iloveloopers** [comment]
Based jamie fr
